cluster/gce/manifests/kube-addon-manager.yaml

```diff
@@ -30,13 +30,21 @@ spec:
     - mountPath: /var/log
       name: varlog
       readOnly: false
+    - mountPath: /etc/srv/kubernetes/
```

Comment:
Probably want to make this `/etc/srv/kubernetes/addon-manager/` as below?

Reply:
Done.